.github/workflows/release.yaml

```diff
@@ -0,0 +1,64 @@
+name: Release ui-plugin-operator
+on:
+  push:
+    tags:
+      - v*
+  workflow_dispatch:
+    inputs:
+      imageTag:
+        description: Base Image Tag (before the :)
+        type: string
+        required: false
+        default: rancher/ui-plugin-operator
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.22'
```

Comment:
The code is currently using go1.19, I think we should use the same version here to avoid any possible problems